.travis.yml

```diff
@@ -27,6 +27,8 @@ stages:
   script:
     - php .Build/bin/phpunit -c ./phpunit.xml --coverage-clover=coverage.xml
     - bash <(curl -s https://codecov.io/bash)
+  before_script:
+    - if [ "$dependencies" = "lowest" ]; then composer update --prefer-lowest --prefer-stable -n; fi;
```

Comment:
Can we move this command to `install`? I think this will improve the reading flow.

Reply:
Moved it to install in commit https://github.com/cvc-digital/typo3-webauthn/pull/8/commits/ad1c8c436c3c2c3ca8cc5596cd4cbbef3a62e03e